metrics/metrics.go

```diff
@@ -98,5 +98,5 @@ func RootScope(i ScopeInit) (tally.Scope, tally.StatsReporter, io.Closer) {
 		}
 		return scope, reporter, closer
 	}
-	return nil, nil, nil
+	return tally.NoopScope, tally.NullStatsReporter, nil
```

Comment:
Good point. However there is no default noop closer and tally does not provide a noop closer. I have one in testutils that I created for unit tests.
It seems contrived to create a Noop closer for this purpose in uberfx. We return nil in a lot of places and don't create noop object everytime.